templates/zerver/integrations/development/dev_panel.html

```diff
@@ -1,99 +1,115 @@
-                    {% for integration in integrations %}
-                    <option value="{{ integration }}"> {{ integration }} </option>
-                    {% endfor %}
-                </select>
-            </td>
-        </tr>
-
-        <tr>
-            <td><b>Fixture</b></td>
-            <td><select id="fixture_name"></select></td>
-        </tr>
-
-        <tr>
-            <td colspan="2" class="pad-top">
-                <label for="URL"><b>URL</b></label>
-                <input id="URL" type="text" />
-            </td>
-        </tr>
-
-        <tr>
-            <td colspan="2" class="center-text pad-top"><b>Custom HTTP Headers</b></td>
-        </tr>
-        <tr>
-            <td colspan="2" class="form-group">
-                <textarea id="custom_http_headers" class="form-control"></textarea>
-            </td>
-        </tr>
-
-        <tr>
-            <td colspan="2" class="center-text pad-top"><b>JSON Body</b></td>
-        </tr>
-        <tr>
-            <td colspan="2" class="form-group">
-                <textarea id="fixture_body" class="form-control"></textarea>
-            </td>
-        </tr>
-
-        <tr>
-            <td colspan="2">
+<div class="pad-small"></div>
+
+<div id="dev-panel">
+    <div class="row1">
+        <div>
+            <label for="bot_name"><b>Bot</b></label>
+            <select id="bot_name">
+                <option value=""></option>
+                {% for bot in bots %}
+                <option value="{{ bot.api_key }}"> {{ bot.full_name }} </option>
+                {% endfor %}
+            </select>
+        </div>
+
+        <div>
+            <label><b>Integration</b></label>
+            <select id="integration_name">
+                <option value=""></option>
+                {% for integration in integrations %}
+                <option value="{{ integration }}"> {{ integration }} </option>
+                {% endfor %}
+            </select>
+        </div>
+
+        <div>
+            <label for="fixture_name"><b>Fixture</b></label>
+            <select id="fixture_name"></select>
+        </div>
+
+        <div>
+            <label class="optional"><b>Stream</b></label>
+            <input id="stream_name" type="text" />
+        </div>
+
+        <div>
+            <label class="optional"><b>Topic</b></label>
+            <input id="topic_name" type="text" />
+        </div>
+    </div>
+
+    <br>
+
+    <div class="row2">
+        <label for="URL"><b>URL</b>  (Automatically Generated)</label>
+        <input id="URL" type="text" />
+    </div>
+
+    <br>
+
+    <div class="row3">
+        <label for="custom_http_headers"><b>Custom HTTP Headers</b></label>
+        <textarea id="custom_http_headers"></textarea>
+    </div>
+
+    <br>
+
+    <div class="row4">
+        <div class="col1">
+            <div class="inline">
+                <label for="fixture_body"><b>Fixture Body</b></label>
                 <p id="message"></p>
```

Comment:
I changed this to a `<span>` to fix a styling glitch where the two big textboxes on the bottom wouldn't be aligned when this is visible.  `span` is what you mean here anyway; it's not a paragraph that needs inter-paragraph margins.